homeassistant/components/mystrom/switch.py

```diff
@@ -72,6 +73,14 @@ def available(self):
         """Could the device be accessed during the last update call."""
         return self._available
 
+    @property
+    def device_state_attributes(self):
+        """Return the device state attributes."""
+        if self.plug.temperature is not None:
+            self._attrs[ATTR_TEMPERATURE] = self.plug.temperature
```

Comment:
We should publish this as a sensor, not include it as an attribute of the switch.